src/main/kotlin/no/uio/microobject/main/MainKt.kt

```diff
@@ -78,6 +88,8 @@ class Main : CliktCommand() {
         "--load" to "repl",       "-l" to "repl",
     ).default("repl")
 
+    private val reasoner by option("--jenaReasoner", "-j", help="Set value of reasoner to 'off', 'rdf', or 'owl' (defaut 'owl')").default("owl")
```

Comment:
1. Please update the help text to describe that this is referring to the internally used reasoner. 
2. defaut -> default